analyses/chromothripsis/04-plot-chromothripsis-and-breakpoint-data.Rmd

```diff
@@ -88,9 +88,11 @@ merge <- merge %>%
 ```{r}
 # Fit linear regression and create plot label
 fit <- lm(sv_breaks_count ~ cnv_breaks_count, data = merge)
+corr <- sqrt(summary(fit)$adj.r.squared)
+pval <- summary(fit)$coef[2,4]
```

Comment:
Instead of using this for the labeling, I'd recommend that we use `cor.test()`, since this will avoid having to "pick" between the two R^2 value returned by `lm`.

```suggestion
result <- cor.test(merge$sv_breaks_count ~ merge$cnv_breaks_count)
corr <- result$estimate
pval <-  result$p.value
```

Reply:
Looks good - I edited slightly and committed!

Reply:
It looks like the R value did change slightly though (0.436-->0.438) so we should probably make sure all the plots get updated - I re-ran `04-plot-chromothripsis-and-breakpoint-data.Rmd` but not figure generation (I figured you might be re-running that anyway)

Reply:
Yep, I'll re-run this.